lib/projects/localDev/AppDevModeInterface.ts

```diff
@@ -87,7 +88,12 @@ class AppDevModeInterface {
     const app = results.find(app => app.sourceId === this.appNode?.uid);
 
     if (!app) {
-      throw new Error('Unable to find app in the project account');
+      throw new Error(
```

Comment:
We could also just log an error here and exit the process

Reply:
https://github.com/HubSpot/hubspot-cli/pull/1532/commits/7b53a192bf602653575cdc26dde0fb3eadecbf2c